invenio_rdm_records/services/github/metadata.py

```diff
@@ -68,13 +68,14 @@ def default_metadata(self):
 
         return dict(
             description=self.description,
-            rights={"id": default_right},
+            rights=[{"id": default_right}],
             publication_date=self.rdm_release.release_payload["published_at"][:10],
             related_identifiers=[self.related_identifiers],
             version=version,
             title=self.title,
             resource_type={"id": "software"},
             creators=self.contributors,
+            publisher="GitHub",
```

Comment:
Note: not sure about the value: in the deposit form publisher in being entered by user

Reply:
we have a config for this as default https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/config.py#L772